doc/classes/EditorSyntaxHighlighter.xml

```diff
@@ -22,5 +22,11 @@
 				Virtual method which can be overridden to return the supported language names.
 			</description>
 		</method>
+		<method name="get_edited_resource">
+			<return type="RefCounted" />
+			<description>
+				Returns the associated [Resource].
```

Comment:
This documentation needs work. Why mention `Resource` if the return type is `RefCounted`? How does this interact with custom resource loader plugins? Can it be null? What happens if there is no resource (text file)?

Reply:
I tried to find equivalent language in other files, but in my haste i did not. I honestly couldnt figure out would would be the correct language so just went with whatever seemed right at the time.

Reply:
how about "Returns a RefCounted Resource if text was loaded from a resource, or else null."

I believe resource loaders are checked long before we get to this. There is a whole separate path for resource loaders, with imports and such.